tools/scripts/firehose.py

```diff
@@ -61,11 +61,15 @@
 
 class YieldedRows(StringIO):
 
-    def __init__(self, job_id, rows, *args, **kwargs):
+    def __init__(self, job_id, rows, time_delta, *args, **kwargs):
         self.rows = rows
+        created_time = datetime.datetime.today() - datetime.timedelta(days=time_delta[0], hours=time_delta[1], seconds=5)
```

Comment:
It might be worth using `timeit` to see what the cost is for running _this_ code 1B times, since that was one of our original use cases here - how much additional timing/overhead does it cost us to generate these strings 1B times?

Reply:
ah I can take that out of there and do it elsewhere